src/app/create/basicauction/page.tsx

```diff
@@ -0,0 +1,207 @@
+        <div className={S.count}>{fields.length}/10</div>
+        <div className={S.imageBoard}>
+          <ul className={S.imageList}>
+            {fields.map(({ imageRequiredId, file }, index) => (
+              <li key={imageRequiredId} className={S.image}>
+                <img src={URL.createObjectURL(file)} alt={URL.createObjectURL(file)} />
+                <button type="button" className={S.deleteButton} onClick={() => remove(index)}>
+                  <DeleteIcon />
+                </button>
+              </li>
+            ))}
+          </ul>
+          <div className={S.imageUpload}>
+            <label htmlFor="image" className={S.imageInfo}>
+              이미지 등록하기
+              <input
+                className={S.imageInput}
+                ref={inputFile}
+                id="image"
+                type="file"
+                accept="image/*"
+                multiple
+                onChange={addImage}
+              />
+            </label>
+          </div>
+        </div>
+        {errors.imageRequired && (
+          <span className={S.error}>
+            <ErrorIcon />
+            {errors.imageRequired.root?.message}
+          </span>
+        )}
+      </div>
+      <label htmlFor="info" className={S.auctionLabel}>
+        <span className={S.title}>상품 정보</span>
+        <div className={S.count}>
+          {watch('infoRequired') ? watch('infoRequired').length : 0}/500
+        </div>
+        <textarea
+          id="info"
+          className={S.info}
+          maxLength={500}
+          {...register('infoRequired', {
+            required: '상품 정보를 입력해 주세요.',
+            validate: {
+              minLength: value => value.length >= 10 || '최소 10글자 이상 작성해야 합니다.',
+            },
+          })}
+        />
+        {errors.infoRequired && (
+          <span className={S.error}>
+            <ErrorIcon />
+            {errors.infoRequired?.message}
+          </span>
+        )}
+      </label>
+      <div className={S.auctionLabel}>
+        <div className={S.title}>경매 기간</div>
+        <div className={S.period}>
+          <label htmlFor="endDate" className={S.subTitle}>
+            종료일
+            <input
+              id="endDate"
+              type="date"
+              {...register('dateRequried', {
+                required: '종료일을 입력해 주세요.',
+                validate: value => {
+                  const currentDate = new Date().toISOString().split('T')[0]; // 현재 날짜 추출
+                  return value >= currentDate || '종료일은 현재 현재 날짜 이후여야 합니다.';
+                },
+              })}
+            />
+            {errors.dateRequried && (
+              <span className={S.error}>
+                <ErrorIcon />
+                {errors.dateRequried?.message}
+              </span>
+            )}
+          </label>
+          <label htmlFor="endTime" className={S.subTitle}>
+            종료 시간
+            <input
+              id="endTime"
+              type="time"
```

Comment:
```suggestion
              id="endTime"
              className={S.auctionInput}
              type="time"
```
여기두 위쪽의 input이랑 같은 디자인을 주는 것은 어떤가요?
종료일도 비슷하게 줘도 좋을 것 같아요

Reply:
넵 좋은 것 같아요! 수정했습니다. [f258ce9](https://github.com/Kernel360/F2-omocha-FE/pull/24/commits/f258ce96c9d916b1efebdc4649ca3dfcb36ede03)